keyboards/leafcutterlabs/bigknobv2/bigknobv2.c

```diff
@@ -0,0 +1 @@
+#include "bigknobv2.h"
```

Comment:
Need a license header here.

Reply:
@noroadsleft  - I didn't write this code, so I don't know what license the original author would use. What license should I use?
https://github.com/LeafCutterLabs/bigKNOB/blob/master/Software/QMKv2/bigKNOBv2.c

Reply:
My instinctive idea is to assign it to `Copyright 2020 LeafCutterLabs, loganmarchione` since you've taken their code and adapted/updated it.